WcaOnRails/config/locales/it.yml

```diff
@@ -1634,7 +1701,7 @@ it:
         content:
           #original_hash: 57a3767
           '1': >-
-            Ti viene assegnato un WCA ID quanto i risultati della tua prima
+            Ti viene assegnato un WCA id quanto i risultati della tua prima
```

Comment:
For consistency, we refer to these as "WCA ID"s, not "WCA id"s. Travis won't pass until you fix this: https://travis-ci.org/thewca/worldcubeassociation.org/builds/197084501#L445.